{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/links/record_links.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# My site is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from __future__ import absolute_import, print_function
+
+from flask import url_for, g
+
+from invenio_records_rest.proxies import current_records_rest
+
+
+def record_links_factory(pid, record=None, **kwargs):
```

Comment:
Perhaps this can be added to https://github.com/inveniosoftware/invenio-records-files/blob/master/invenio_records_files/links.py?

Reply:
Ah yes that would indeed make more sense.